examples/script_nifti.py

```diff
@@ -0,0 +1,41 @@
+import os
```

Comment:
It's good form to always start scripts with a "[shebang](https://en.wikipedia.org/wiki/Shebang_(Unix))". And to run `chmod +x examples/script_nifti.py`.

```suggestion
#!/usr/bin/env python3

import os
```

It's only for unix, and `console_scripts` supersedes this anyway, but I think it's a good practice.